src/client/package.json

```diff
@@ -9,16 +9,19 @@
         "@testing-library/jest-dom": "^4.2.4",
         "@testing-library/react": "^9.5.0",
         "@testing-library/user-event": "^7.2.1",
+        "jsonwebtoken": "^8.5.1",
         "lodash": "^4.17.20",
+        "luxon": "^1.26.0",
         "moment": "^2.29.1",
         "react": "^16.13.1",
         "react-dom": "^16.13.1",
         "react-router-dom": "^5.2.0",
-        "react-scripts": "3.4.1"
+        "react-scripts": "3.4.1",
+        "typescript": "^4.2.3"
```

Comment:
didn't see any use of this. why did we add typescript package? 
same question about react-scripts

Reply:
Let me see if I can figure out which package has that as a dependency. Not anything I did explicitly.

Reply:
`react-scripts` was present before my time: https://github.com/CodeForPhilly/paws-data-pipeline/commit/d1b657a723810ce2cbbfcbd133a709794326285e

`typescript` was  not needed., removed in  43e4b0c